### ISSUE TITLE
Surface UI

### ISSUE THREAD

**Issue by nelsonic** [dwyl/technology-stack#97]
mentioned in https://github.com/dwyl/learn-phoenix-framework/issues/145#issuecomment-954295899 deserves its' own issue.
+ [ ] https://surface-ui.org/samplecomponents
+ [ ] https://github.com/surface-ui/surface
+ [ ] ElixirConf 2021 - Dave Lucia - Surface: a bridge to the Javascript community: https://youtu.be/Lh5rA1pgWCk

Need to make time to investigate this after PETAL #87

**Comment by nelsonic** [comment]
@SimonLab do you have time investigate this? (or want to pair on it when you're back?)

**Comment by SimonLab** [comment]
Yes I can have a look and see if the features provided by the library can be useful :+1:

**Comment by nelsonic** [comment]
Merci Simon! 🙌 
Basically we want to know if **`Surface UI`** will "play nice" with **`PETAL`** i.e. ***`PETALS`*** 😜 🤞 
(if not, it's totally fine, we can just crack on with what we have ...)
